vllm_ascend/models/deepseek_v2.py

```diff
@@ -179,6 +180,12 @@ def __init__(
         else:
             self.gate.e_score_correction_bias = None
 
+        self.enable_cv_parallel = False
+        additional_config = get_current_vllm_config().additional_config
```

Comment:
please use ascend_config instead now. Note that doc should be updated at the same time.